src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Square extends Figure {
+    private int side;
```

Comment:
Consider using a more descriptive variable name, such as 'sideLength'. This would make the code more readable and understandable. 📚